src/test/scala/cromwell/MainSpec.scala

```diff
@@ -5,10 +5,11 @@ import java.io.ByteArrayOutputStream
 import akka.actor.ActorSystem
 import akka.testkit.EventFilter
 import com.typesafe.config.ConfigFactory
+import cromwell.binding.WdlExpression
 import cromwell.server.WorkflowManagerSystem
 import cromwell.util.{SampleWdl, FileUtil}
 import cromwell.util.SampleWdl.ThreeStep
-import org.scalatest.{FlatSpec, Matchers}
+import org.scalatest.{BeforeAndAfterAll, FlatSpec, Matchers}
```

Comment:
Bleh, I think I forgot to optimize the imports on this file after removing that function